external/build/fugu-showcase.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * @fileoverview Fetches the Fugu Projects following the data process described
+ * in https://github.com/GoogleChromeLabs/fugu-showcase#data-flow.
+ */
+
+require('dotenv').config();
+
+const {default: fetch} = require('node-fetch');
+const path = require('path');
+const {unfurl} = require('unfurl.js');
+const sharp = require('sharp');
+const fs = require('fs').promises;
```

Comment:
From Node.js v14 on, the Promises API is properly exported and available from `require('fs/promises')`.